io.github.aliasbody.xenia.yml

```diff
@@ -0,0 +1,96 @@
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: xenia.sh
+
+# Support 32-bit at runtime
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '23.08'
+    
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    versions: "23.08;23.3.1"
+    version: '23.3.1'
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+    
+    org.freedesktop.Platform.ffmpeg-full:
+      directory: lib/ffmpeg
+      add-ld-path: .
+      version: '23.08'
+      autodelete: false
+
+  org.freedesktop.Platform.ffmpeg_full.i386:
+      directory: lib32/ffmpeg
+      add-ld-path: .
+      version: '23.08'
+      autodelete: false
+
+# Support 32-bit at buildtime
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --allow=multiarch
+  - --filesystem=/media:ro
+  - --filesystem=/run/media:ro
+  - --filesystem=home
+
+modules:
+  # Create 32-bit directories
+  - name: compat
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/i386-linux-gnu
+      - mkdir -p /app/lib/i386-linux-gnu/GL
+      - mkdir -p /app/bin
+      - mkdir -p /app/lib64
+      - mkdir -p /app/share
+      - mkdir -p /app/lib/ffmpeg
+      - mkdir -p /app/lib32/ffmpeg
+
+  - name: wine-ge-8.25
+    buildsystem: simple
+    build-commands:
+      - cp -a bin/* /app/bin
+      - cp -a lib/* /app/lib
+      - cp -a lib64/* /app/lib64
+      - cp -a share/* /app/share
+    sources:
+      - type: archive
+        url: https://github.com/GloriousEggroll/wine-ge-custom/releases/download/GE-Proton8-25/wine-lutris-GE-Proton8-25-x86_64.tar.xz
+        sha512: 03bc8e5f77a82226cc89d6141be4dd77afad003d23ce4fb98987b4446318c7c68953b5a10c379cbaf042a9185dd4629895e14cd9878b272cf74139ac0c07ee63
+        
+  - name: xenia-v1.0.2808-master
+    buildsystem: simple
+    build-commands:
+      - ls
+      - mkdir -p /app/xenia
+      - cp -a * /app/xenia
+    sources:
+      - type: archive
+        url: https://github.com/xenia-project/release-builds-windows/releases/download/v1.0.2808-master/xenia_master.zip
+        sha512: 4349f5a4307f9d60bc00dfa8dfa9fb146e66acc9acdf22e48d0aab6cd21444f83cc6f72a19f320ea82846377b644419dd2180e37c23cabfde68260ccb98857e9
+        
+  - name: flatpak-xenia
+    buildsystem: simple
+    build-commands:
+      - install -D xenia.sh /app/bin/xenia.sh
+      - install -Dm644 io.github.aliasbody.xenia.desktop /app/share/applications/io.github.aliasbody.xenia.desktop
+      - install -Dm644 xenia-icon.png /app/share/icons/hicolor/128x128/apps/io.github.aliasbody.xenia.png
+    sources:
+      - type: git
+        url: https://github.com/aliasbody/flatpak-xenia
```

Comment:
why aren't these part of upstream?

Reply:
Yeah, I'm also confused. Also it sounds like the ID should be `jp.xenia.Xenia`?

Reply:
> Yeah, I'm also confused. Also it sounds like the ID should be `jp.xenia.Xenia`?

It's what makes sense to me as well, but since I'm not the author of Xenia, it isn't managed by them (for now) and I created a github just for it with the missing components, I didn't knew if I could use the jp.xenia.Xenia. But if yes then I will change it

Reply:
The best would be to sort this out with upstream developers. We won't accept a Wine-based app if it's not going to be officially maintained.